packages/util/src/constants.ts

```diff
@@ -14,14 +14,27 @@ export const MAX_INTEGER = new BN(
   16
 )
 
+/**
+ * The max integer that the evm can handle (2^256-1) as a bigint
+ */
+export const MAX_INTEGER_BIGINT = 2n ** 256n - 1n
+
 /**
  * 2^256
+ *
+ * @deprecated
```

Comment:
maybe along with a note
```suggestion
 * @deprecated bn.js constants are deprecated, please use the newly introduced bigint constants
```